elyra/__init__.py

```diff
@@ -49,8 +49,11 @@ def _load_jupyter_server_extension(nb_server_app):
         (url_path_join(web_app.settings['base_url'], r'/elyra/namespace'), NamespaceHandler),
         (url_path_join(web_app.settings['base_url'], r'/elyra/pipeline/schedule'), PipelineSchedulerHandler),
         (url_path_join(web_app.settings['base_url'], r'/elyra/pipeline/export'), PipelineExportHandler),
+        (url_path_join(web_app.settings['base_url'], r'/elyra/pipeline/config/%s' % (resource_regex)),
+         PipelineConfigHandler),
```

Comment:
This endpoint should be added to the [elyra.yaml](https://github.com/elyra-ai/elyra/blob/master/elyra/api/elyra.yaml) open api doc.

Reply:
As I mentioned, this part is the one I would like to hold off until the "component registry" is done.